randovania/server/async_race/room_api.py

```diff
@@ -369,12 +370,15 @@ def join_and_export(sa: ServerApp, room_id: int, auth_token: str, cosmetic_json:
     preset = layout_description.get_preset(players_config.player_index)
     cosmetic_patches = preset.game.data.layout.cosmetic_patches.from_json(cosmetic_json)
 
+    # Overwrite spoiler property, as some games rely on this to provide another alternative to show spoilers.
+    layout_description = dataclasses.replace(
```

Comment:
This changes the permalink, so this is wrong.

Reply:
oh. i'll come up with an alternative then.